src/containers/pages/JurisdictionMetadata/index.tsx

```diff
@@ -128,6 +133,14 @@ const JurisdictionMetadataImportView = () => {
     submitForm: formSubmit,
   };
 
+  /** props for the Jurisdiction Hierarchy form */
+  const jurisdictionMetadataDownloadFormProp: JurisdictionHierachyDownloadFormProps | any = {
```

Comment:
If you remove the `any` here you'll see that this object does not conform to the `JurisdictionHierachyDownloadFormProps`  interface.

Reply:
addressed via https://github.com/onaio/reveal-frontend/pull/1062/commits/b61f7c5b3bdc3ee4ab8930041d3a9a864cc5fbbb